dev-lang/icon/icon-9.5.1-r3.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="very high level language"
```

Comment:
`Very`

Reply:
I'll fix these and push a new commit.

BTW.. I've worked with upstream and we came up with a way of handling the upstream repo.  Upstream will be tagging their repo with a running version number when there are bug fixes.  I'm working on a new ebuild that will take advantage of the tags, so expect a new commit with a new version number soon.

Some quick background, the language itself has been stabalized and additional upstream work will probably be limited to bug fixes or fixing issues found with new levels of the supported C compilers, etc.

I'm also looking at creating an ebuild for unicon (a OO version of Icon that is still in active development).